Source/SwiftLintFramework/Rules/Style/FileTypesOrderRuleExamples.swift

```diff
@@ -130,6 +118,17 @@ internal struct FileTypesOrderRuleExamples {
         extension Foo {}
         extension Bar {
         }
+        """,
+        """
```

Comment:
Due to #3040, please wrap all new examples in `Example(...)`.

Reply:
Thanks for the pointer, fixed via rebase to solve merge conflicts (where I also fixed this).